build/pipelines/templates/helix-processtestresults-job.yml

```diff
@@ -7,7 +7,7 @@ parameters:
 
 jobs:
 - job: ProcessTestResults
-  condition: succeededOrFailed()
+  condition: ${{ parameters.condition }}
```

Comment:
Since you are using a parameter here, I would list it above in the parameters, and give it a default value.

Reply:
OK.